commons/all/src/main/java/org/infinispan/commons/util/ObjectDuplicator.java

```diff
@@ -20,8 +20,6 @@
 public class ObjectDuplicator {
```

Comment:
This class can be removed. It is deprecated and used only in a handful of tests.

Reply:
Removals of deprecated things will happen in separate PRs, such as https://github.com/infinispan/infinispan/pull/14771